cmd/client-s3.go

```diff
@@ -207,10 +207,10 @@ func (n notifyExpiringTLS) RoundTrip(req *http.Request) (res *http.Response, err
 
 // getTransportForConfig returns a corresponding *http.Transport for the *Config
 // set withS3v2 bool to true to add traceV2 tracer.
-func getTransportForConfig(config *Config, withS3v2 bool) http.RoundTripper {
+func getTransportForConfig(config *Config, withS3v2 bool, chainsTLS bool) http.RoundTripper {
```

Comment:
Why is this variable named `chainsTLS`? I guess `forceTLS` would be a better name, because it forces a TLS transport.

Reply:
No, if you set `forceTLS` to `false`, then it may/may not be TLS. If it's set to `true` then you're actually forcing it to `true`. I'm not sure what `chainsTLS` means, but I don't think it's suitable in this case. You typically use _chains_ with a list of TLS certificates to the CA root. That's called a _chain_, but that's not what is being used here.